specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -9233,9 +9234,14 @@
           "description": "Value for Constant/Field Type: string.",
           "type": "string"
         },
-        "operator": {
-          "description": "Expression operator value Type: string.",
-          "type": "string"
+        "operators": {
+          "description": "Expression operator value Type: list of strings.",
+          "type": "array",
+          "items": {
+            "type": "object",
```

Comment:
> "type": "object", [](http://example.com/codeflow?start=11&length=18)

can u make this type: string?

Reply:
there are few instances where "type": "object" is used when the items are strings [https://github.com/Azure/azure-rest-api-specs/blob/1d103acc2539e143e3ef679a1a2d88ef66c357a4/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json#L8962C11-L8967C13](url) Is it a general recommendation to have "type":"string" or wouldn't it be safe to use "type": "object"?